virttest/utils_misc.py

```diff
@@ -840,6 +840,12 @@ def archive_as_tarball(source_dir, dest_dir, tarball_name=None,
     >>> utils_misc.archive_as_tarball('/tmp', '/net/server/backup',
                                       'host1-tmp', 'gz')
     '''
+    # Refer to https://docs.python.org/3/library/tarfile.html#tarfile.open
+    supported_compression = ['gz', 'bz2', 'xz']
+    if compression not in supported_compression:
+        raise exceptions.TestError("compression %s is not supported method %s"
+                                   % (compression, supported_compression))
+
```

Comment:
@clebergnu 
Thanks for your reminder.
I have updated it and Please recheck it.